python/ray/experimental/state.py

```diff
@@ -1310,9 +1310,19 @@ def cluster_resources(self):
 
         return dict(resources)
 
+    def _live_client_ids(self):
+        """Returns a set of client IDs corresponding to clients still alive."""
```

Comment:
Does this actually work? I thought you can get an isinsertion and then a deletion.

Reply:
we removed that behavior in #2880